.devcontainer/develop_on_containers.rst

```diff
@@ -0,0 +1,67 @@
+.. _develop_on_remote_containers:
+
+Develop on a remote container
+=============================
+
+.. note:: This approach requires to have a docker image with MAPDL installed.
+   The one listed in the 
+   `repository development container configuration files <pymapdl_devcontainer_configuration_>`_ 
+   is not for public usage due to licensing issues.
+   However, you are invited to `create your own docker image <ref_make_container_>`
+   and use this file as template for setting your own local development container.
+
+To use a `remote container <vscode_devcontainers_containers_>`_, you must install:
+
+* `VS Code <vscode_>`_
```

Comment:
```suggestion
* `Visual Studio Code <vscode_>`_
```
Now I'm more confident that all the occurrences of Visual Code should be Visual Studio Code. We shouldn't abbreviate product names.